### ISSUE TITLE
Add thumbnails in the list file view (Addition to the new filesystem)

### ISSUE THREAD

**Issue by Keetz** [godotengine/godot#22303]
**Issue description:**
We have thumbnails in the new tree view, it would be nice to have thumbnails in the list file view as well.

> [14:20:49] <Keetz> Groud: just an idea, nothing to be done now, but would it make sense to have thumbnails in the list file view too?
> [14:25:48] <Groud> Keetz: hmm
> [14:25:52] <Groud> I did not think about that
> [14:26:04] <Groud> But yeah it makes sense, and should not be difficult to add
> [14:27:05] <Groud> Keetz: Once this is merged, please open an issue an mention me ;)
> [14:27:54] <Keetz> Groud: will do ;)

@groud

**Comment by groud** [comment]
>  Once this is merged, please open an issue an mention me

A job perfectly done ! ^^
